delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/stream/ProjectionStore.scala

```diff
@@ -73,12 +73,12 @@ trait ProjectionStore {
     * @param failures
     *   the FailedElem to save
     */
-  def saveFailedElems(metadata: ProjectionMetadata, failures: List[FailedElem]): UIO[Unit]
+  def saveFailedElems(metadata: ProjectionMetadata, failures: List[FailedElem])(implicit clock: Clock[UIO]): UIO[Unit]
```

Comment:
The clock should be already provided by the implementation

Reply:
Fixed